.cspell.json

```diff
@@ -384,6 +384,7 @@
     "burlywood",
     "butterworth",
     "buttord",
+    "ByteTrack",
```

Comment:
@miursh I think `ByteTrack` is split into `Byte` and `Track` (normal words) when the spelling is checked.
So you meant `bytetrack`?

```suggestion
    "bytetrack",
```

Reply:
Thanks. Either is ok for me.

Reply:
@miursh Where did you encounter this error? Could you please give me the link?

Reply:
Thank you. Okay then `bytetrack` is good!